### PR TITLE
Chapter 7 -- new repo

### PR DESCRIPTION
**Pull request by laumurph**

### CODE REVIEW

File: _sources/Conditionals/intro-TurtlesandConditionals.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+.. qnum::
+   :prefix: condition-1-
+   :start: 1
+
+Intro: What we can do with Turtles and Conditionals
+---------------------------------------------------
+
+To build further on our turtle drawings, we can start to include something called a **selection** or 
+**conditional statement**. Using this kind of statement will allow up to check conditions and change the
```
Comment (bnmnetp) [inline _sources/Conditionals/intro-TurtlesandConditionals.rst]: typos:  or a **conditional statement**  (insert a)

all us to check (not up)

File: _sources/Conditionals/intro-TurtlesandConditionals.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+To build further on our turtle drawings, we can start to include something called a **selection** or 
+**conditional statement**. Using this kind of statement will allow up to check conditions and change the 
+behavior of the program accordingly. 
+
+.. activecode:: ac7_1_1
+
+    import turtle
+    wn = turtle.Screen()
+
+    Amy = turtle.Turtle()
```
Comment (bnmnetp) [inline _sources/Conditionals/intro-TurtlesandConditionals.rst]: Fix capitalization -- I get that the names are proper nouns, but I am a stickler on following the convention that variables should be lower case while classes are Capitalized.

File: _sources/Conditionals/Theinandnotinoperators.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: condition-4-
+   :start: 1
+
+The ``in`` and ``not in`` operators
```
Comment (bnmnetp) [inline _sources/Conditionals/Theinandnotinoperators.rst]: I think we need to add the list examples to this section, it keeps the parallelism we've been using with exposing sequence stuff.

File: _sources/Conditionals/Chainedconditionals.rst [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+    if x < y:
+        print("x is less than y")
+    elif x > y:
+        print("x is greater than y")
+    else:
+        print("x and y must be equal")
+
+
+.. note::
```
Comment (bnmnetp) [inline _sources/Conditionals/Chainedconditionals.rst]: Can remove this

File: _sources/Conditionals/Nestedconditionals.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+            print("x and y must be equal")
+
+.. note::
+
+    In some programming languages, matching the if and the else is a problem. However, in Python this is not 
+    the case. The indentation pattern tells us exactly which else belongs to which if.
+
+If you are still a bit unsure, here is the same selection as part of a codelens example.  Step through it to see how the correct ``print`` is chosen.
+
+.. codelens:: clens7_8_1
```
Comment (bnmnetp) [inline _sources/Conditionals/Nestedconditionals.rst]: Something is funny here, after the first line I lose the red arrow...

Comment (laumurph) [reply _sources/Conditionals/Nestedconditionals.rst]: Huh, that doesn't happen when I run the page for some reason.

Comment (bnmnetp) [reply _sources/Conditionals/Nestedconditionals.rst]: OK, I'll look into it more on my end.

Comment (laumurph) [reply _sources/Conditionals/Nestedconditionals.rst]: I think I'm using the github version of runestone - I don't recall using pypi.

Comment (bnmnetp) [reply _sources/Conditionals/Nestedconditionals.rst]: If you did ``pip install runestone`` then you are using the released pypi version.  If you did ``pip install -e /path/to/components`` . Then you are using github...  If you do pip list and you see runestone version 3.0.9 with a path next to it that also shows you are using github.  If no path then standard release.

Comment (laumurph) [reply _sources/Conditionals/Nestedconditionals.rst]: I'm seeing version 3.0.7 with a path!

Comment (bnmnetp) [reply _sources/Conditionals/Nestedconditionals.rst]: Then I think you haven't done a git pull into that path for a while!

Comment (bnmnetp) [reply _sources/Conditionals/Nestedconditionals.rst]: When I have a CodeLens it shows up like this:

screen shot 2018-06-07 at 1 13 08 pm

Notice that the last button is below, this does not happen on other versions.  Are you seeing the same?

Comment (laumurph) [reply _sources/Conditionals/Nestedconditionals.rst]: Nope, in my view the last button is in line with the rest of the buttons.

Comment (bnmnetp) [reply _sources/Conditionals/Nestedconditionals.rst]: Wow, crazy.  Could you tar and gzip your source fopp folder and email it to me?  I want to check to see if I can find something different between them.  I've tried a clean check and a build with the released version of runestone and I still keep seeing this weird problem.

**Pull request merged**